msmbuilder/cluster/agglomerative.py

```diff
@@ -30,21 +27,29 @@
 
 __all__ = ['_LandmarkAgglomerative']
 
+
+def ward_pooling_function(x, cluster_cardinality, intra_cluster_sum):
+    normalization_factor = cluster_cardinality*(cluster_cardinality+1)/2
```

Comment:
Yes, https://github.com/brookehus/msmbuilder/blob/c514a2f39ae1b375890ba4c7f77dabbe6707294d/msmbuilder/cluster/agglomerative.py#L10 should take care of it